static/style.css

```diff
@@ -5,9 +5,10 @@
 }
 
 .content {
-	height: 100%;
+	position: absolute;
```

Comment:
Fix indentation. Please don't mix spaces and tabs, use either one.

Reply:
5a10b9a